### PR TITLE
[em] ktest hack

### PR DESCRIPTION
**Pull request by daejunpark**

### CODE REVIEW

**Comment by daejunpark** [comment]
@cos please review.

This is a hack for negative tests of kompilation, exploiting `posixInitScript` to compare outputs of kompile. Note that ktest currently doesn't support it; it doesn't compare outputs of kompile, but check its exit code only.

I'll open an issue for this.

File: k-distribution/tests/regression/error-messages/unsupported-io-unblock-patterns/b.k.out [hunk 1]
```diff
@@ -1,24 +1,12 @@
...
-	Source(./b.k)
-	Location(25,10,25,36)
 [Warning] Compiler: Unsupported matching pattern in stdin stream cell.
 The currently supported pattern is: <in> ListItem(V:Sort) => .List ... </in>
-	Source(./b.k)
-	Location(28,10,28,66)
 [Warning] Compiler: Unsupported matching pattern in stdin stream cell.
 The currently supported pattern is: <in> ListItem(V:Sort) => .List ... </in>
-	Source(./b.k)
-	Location(31,10,31,75)
```
Comment (cos) [inline k-distribution/tests/regression/error-messages/unsupported-io-unblock-patterns/b.k.out:20]: why is the location removed?

**Comment by cos** [comment]
also, please mark this with the right label, milestone, and reviewer :)

File: k-distribution/tests/regression/error-messages/unsupported-io-unblock-patterns/b.k.out (hunk 1 above)
Comment (daejunpark) [inline k-distribution/tests/regression/error-messages/unsupported-io-unblock-patterns/b.k.out:20]: Jenkins spoils the source information, which becomes hard to compare. The file paths are output in an absolute path in Jenkins machine. Also, the line numbers are added +1 due to #1883.

Comment (cos) [inline k-distribution/tests/regression/error-messages/unsupported-io-unblock-patterns/b.k.out:20]: could you display at least the Location?
if it's in any way hard, nevermind

Comment (cos) [inline k-distribution/tests/regression/error-messages/unsupported-io-unblock-patterns/b.k.out:20]: don't worry about the bug, the assertions will be updated when the bug is fixed

Comment (daejunpark) [inline k-distribution/tests/regression/error-messages/unsupported-io-unblock-patterns/b.k.out:20]: Location will be added back once #1883 is fixed. I opened an issue #1984 to keep track of it.

**Comment by daejunpark** [comment]
#1945

**Pull request merged**